controllers/control_plane_init_locker.go

```diff
@@ -117,7 +125,12 @@ func (l *controlPlaneInitLocker) Release(cluster *clusterv2.Cluster) bool {
 }
 
 func (l *controlPlaneInitLocker) configMapExists(namespace, name string) (bool, error) {
-	_, err := l.configMapClient.ConfigMaps(namespace).Get(name, metav1.GetOptions{})
+	// _, err := l.configMapClient.ConfigMaps(namespace).Get(name, metav1.GetOptions{})
```

Comment:
why is this commented out? I think deleting this would be preferred

Reply:
will do.